### PR TITLE
New Presence: Tidal

### LINKED ISSUE #1474: Tidal | tidal.com
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [x] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** Destiny#2923

**What's the name of the service?**  https://tidal.com/ | Tidal

**What should the Presence display?**  Song, Album, Artist, Playing/Paused

**If possible, please provide a logo for the service (512x512 minimum)** 
https://cdn.worldvectorlogo.com/logos/tidal.svg

### PR DESCRIPTION
**Pull request by ysrylmz32**
closes #1474 

Discord_5TtMajnMV8
Discord_g6dnr4kMdv

### CODE REVIEW

File: Tidal/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "description": {
+    "tr": "Tidal yüksek ses kalitesi ve ucuz fiyatlarla hizmet sağlayan bir müzik sitesidir",
+    "en": "Tidal is a music stream service with very high sound quality"
+  },
+  "service": "Tidal",
+  "version": "1.0.0",
+  "logo": "https://who.is-inside.me/pXP8PRGK.jpg",
+  "thumbnail": "https://who.is-inside.me/5IJ3K6fe.png",
+  "color": "#000000",
+  "tags": [
```
Comment (Bas950) [inline Tidal/dist/metadata.json:19]: set a color

Comment (ysrylmz32) [reply Tidal/dist/metadata.json:19]: Changed the icons and the color of service is black ?

Comment (Bas950) [reply Tidal/dist/metadata.json:19]: oh okay nvm then about the color xD

Comment (ysrylmz32) [reply Tidal/dist/metadata.json:19]: so won't you approve xd

Comment (Bas950) [reply Tidal/dist/metadata.json:19]: update the small image keys? Or upload proof that you did so

Comment (ysrylmz32) [reply Tidal/dist/metadata.json:19]: ntJC0i9j80
IiXSthek8S

**Pull request merged**